pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
A string with critical error can be fixed by providing a new right translation. A suggestion, independently of it being right or not, won't fix anything. A suggestion being accepted counts as a new translation, either right or not. So only users with translate or review right can fix a critical error.